index.rst

```diff
@@ -99,7 +155,7 @@ Selecting the Default Configuration
 -----------------------------------
 
 In the high majority of cases, users will want to load the default configuration.
-The default configuration consists of parameters in the `_init.yaml` file and subsequently the `_<site>.yaml`, if it is present.
+The default configuration consists of parameters in the ``_init.yaml`` file and subsequently the ``_<site>.yaml``, if it is present.
```

Comment:
Please consider changing "In the high majority of cases" to "In most cases" or "Operators will almost always load...".